GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GitFilesTests.cs

```diff
@@ -38,10 +38,35 @@ public void CreateFileTest()
             this.fileSystem.CreateEmptyFile(this.Enlistment.GetVirtualPathTo(emptyFileName));
             this.Enlistment.WaitForBackgroundOperations().ShouldEqual(true, "Background operations failed to complete.");
             GVFSHelpers.ModifiedPathsShouldContain(this.fileSystem, this.Enlistment.DotGVFSRoot, emptyFileName);
-            this.Enlistment.GetVirtualPathTo(fileName).ShouldBeAFile(this.fileSystem);
+            this.Enlistment.GetVirtualPathTo(emptyFileName).ShouldBeAFile(this.fileSystem);
         }
 
         [TestCase, Order(2)]
+        public void CreateHardLinkTest()
+        {
+            if (!this.fileSystem.SupportsHardlinkCreation)

```

Comment:
This test is not getting any coverage on Mac as it's not using BashRunner

Reply:
Test fixture updated to use all runners (whatever used to cause failures with `BashRunner` is no longer an issue)